README.md

```diff
@@ -375,8 +375,9 @@ Publication Review - Publish)
   # run the output queue processing job
   heroku run -s performance-m rails dss:process_output_queue --app TARGET-HEROKU-APP
 
-  # wait for all ETD emails to be received (the preservation email is the final one to look for)
-  # scale the worker back down so we do not pay for more CPU/memory than we need
+  # wait for all ETD emails to be received (there are three emails: one overall results summary, one preservation
+  # results summary, and one MARC batch export. The batch export email is typically the final one to look for).
```

Comment:
I'm glad you are updating these docs as I noted the last time I ran them there was a discrepancy but wasn't sure if it was going to be predictive. It seems like it is. That said, would you be okay with removing the bit that says which one is last and just introducing this new change that list what the 3 are? i.e,. just removing `The batch export email is typically the final one to look for` and leaving the rest of this change?

Reply:
If you want to keep that in I'm fine with it though as it does seem to have been stable over the last several runs.

Reply:
Yeah, that works for me.